src/app/admin/scoring/page.jsx

```diff
@@ -29,14 +35,13 @@ export default function AuthAdminUserPeek({ params: asyncParams }) {
 
 function AdminScoring({ imageConfig }) {
   const stats = useAdminStats();
-  // const allMovies = useAdminAllMovies({ limit: 250 });
 
   if (!stats) {
     return null;
   }
 
   return (
-    <div className="flex flex-col">
+    <div className="flex flex-col max-w-full">
```

Comment:
```suggestion
    <div className="flex flex-col w-full">
```
Would be nice if the table didn't change width as you applied filters.

Reply:
Ah good call.